icons/android/back-36.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="36dp"
+    android:height="36dp"
+    android:viewportWidth="36"
+    android:viewportHeight="36">
+    <path
+        android:fillColor="#0C0C0D"
+        android:fillAlpha=".8"
```

Comment:
@aminalhazwani The fillAlpha in the 24dp assets was creating issues matching the colors specified in the specs: is there any reason it's included here? fwiw, we removed the `fillAlpha` from the 24dp assets in components.

Reply:
Relatedly, any changes we make in components should probably be reflected here, upstream, right? Do you have any ideas on how we can make sure that happens?

Reply:
@mcomella specifically to connect you can set the `fillColor` to Grey 10 with no `fillAlpha`

Reply:
> @mcomella specifically to connect you can set the `fillColor` to Grey 10 with no `fillAlpha`

I removed `fillAlpha` and tinted it in code (rather than changing the assets).

> @mcomella I think is there because of how the `svg2vectordrawable` package works, I mentioned something similar here [#35 (comment)](https://github.com/FirefoxUX/photon-icons/pull/35#discussion_r210206664)

Is the `fillAlpha` desirable? Assuming you execute this on the command line, I could quickly write you a wrapper script that would remove `fillAlpha` from the output (using `sed`).

> > Relatedly, any changes we make in components should probably be reflected here, upstream, right? Do you have any ideas on how we can make sure that happens?
> 
> Maybe task cluster? So that every time an update is push a PR/Issue is open?

Filed https://github.com/mozilla-mobile/android-components/issues/817 to come up with a solution.

Reply:
> Is the fillAlpha desirable? Assuming you execute this on the command line, I could quickly write you a wrapper script that would remove fillAlpha from the output (using sed).

@mcomella we have a defined set of fill colors that we currently support, this selection goes across platforms, so not Android-only

- Grey 10
- Grey 10 a80
- Grey 10 a60
- Grey 90 a80
- Grey 90 a60
- Blue 40
- Blue 50
- Green 60
- Red 60
- White 100

Would be interesting to understand how can we enable engineering to pick only from those colors so that they don't have to manually modify assets 🙂

Reply:
I don't think there's an easy way to do this. I filed: https://github.com/mozilla-mobile/android-components/issues/935

Traditionally, we do separate color from asset:
```xml
android:src="@drawable/components_asset
android:tint="@color/photon_grey10"
```